### PR TITLE
Added a friend named Regex

### PR DESCRIPTION
**Pull request by PijusOz**

### CODE REVIEW

File: front-end-side/src/Pages/Friends.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 import Navbar from "../../src/Components/Navbar";
+import "../Pages/Friends.css";
 const Friends = () => {
     return (
         <div>
             <Navbar/>
-            <h1>Friends</h1>
+            <h1>Friends:</h1>
+            <h2>Regex</h2>
+            <img src="/dog.gif" alt=""/>
```
Comment (dissco6) [inline front-end-side/src/Pages/Friends.js:10]: Why alt tag is empty? What will will be shown if image will not be loaded

Comment (dissco6) [reply front-end-side/src/Pages/Friends.js:10]: Is the .gif optimized?

File: front-end-side/src/Pages/Friends.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+body {
+}
```
Comment (dissco6) [inline front-end-side/src/Pages/Friends.css:2]: Why you add body tag if it's empty?

Comment (PijusOz) [reply front-end-side/src/Pages/Friends.css:2]: It was automatically created, I will delete it, sorry.

File: front-end-side/src/Pages/Friends.js (hunk 1 above)
Comment (PijusOz) [reply front-end-side/src/Pages/Friends.js:10]: Thanks for the "alt" reminder. Optimize the gif yourself.

File: front-end-side/src/Pages/Friends.js [hunk 3]
```diff
@@ -1,10 +1,13 @@
 import React from "react";
 import Navbar from "../../src/Components/Navbar";
+import "../Pages/Friends.css";
 const Friends = () => {
     return (
         <div>
```
Comment (dissco6) [inline front-end-side/src/Pages/Friends.js:6]: Change it to <div class="container"> in order to align content right

Comment (PijusOz) [reply front-end-side/src/Pages/Friends.js:6]: I want it alligned to the left.

Comment (dissco6) [reply front-end-side/src/Pages/Friends.js:6]: It has to have this allignment 🤦‍♂️

File: front-end-side/src/Pages/Friends.css [hunk 4]
```diff
@@ -0,0 +1,3 @@
+img{
+    height: 300px;
```
Comment (shilingas) [inline front-end-side/src/Pages/Friends.css:2]: I guess it won't be really responsive on the phone by using px

Comment (PijusOz) [reply front-end-side/src/Pages/Friends.css:2]: Oh no

**Pull request merged**